src/domain/app/i18n/sv.json

```diff
@@ -535,6 +535,7 @@
         "general": "Publicera evenemanget",
         "volunteering": "Publicera volontärarbetet"
       },
+      "buttonSendToPublishing": "Send to publising",
```

Comment:
Should this be in Swedish?

Reply:
Translations: https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/185/commits/9d8ee2af93331ff695811a6a1d684ae18200167c